app/path.go

```diff
@@ -37,13 +37,14 @@ type pathGenOpts struct {
 }
 
 type generatedPath struct {
-	Path           string `json:"path,omitempty"`
-	PathWithPrefix string `json:"path-with-prefix,omitempty"`
-	Type           string `json:"type,omitempty"`
-	Description    string `json:"description,omitempty"`
-	Default        string `json:"default,omitempty"`
-	IsState        bool   `json:"is-state,omitempty"`
-	Namespace      string `json:"namespace,omitempty"`
+	Path           string   `json:"path,omitempty"`
+	PathWithPrefix string   `json:"path-with-prefix,omitempty"`
+	Type           string   `json:"type,omitempty"`
+	Description    string   `json:"description,omitempty"`
+	Default        string   `json:"default,omitempty"`
+	IsState        bool     `json:"is-state,omitempty"`
+	Namespace      string   `json:"namespace,omitempty"`
+	FeatureList    []string `json:"featurelist,omitempty"`
```

Comment:
imho it would be nicer to name it `if-features` to have a clear mapping to the yang naming